src/passes/DeadArgumentElimination.cpp

```diff
@@ -73,16 +76,17 @@ struct DAEFunctionInfo {
   // removed as well.
   bool hasTailCalls = false;
   std::unordered_set<Name> tailCallees;
-  // Whether the function can be called from places that
-  // affect what we can do. For now, any call we don't
-  // see inhibits our optimizations, but TODO: an export
-  // could be worked around by exporting a thunk that
-  // adds the parameter.
-  // This is atomic so that we can write to it from any function at any time
-  // during the parallel analysis phase which is run in DAEScanner.
-  std::atomic<bool> hasUnseenCalls;
-
-  DAEFunctionInfo() { hasUnseenCalls = false; }
+  // Whether the function can be called from places that affect what we can do.
```

Comment:
It's surprising that `hasUnseenCalls` is a set after reading this comment. I still would have expected it to be a `bool`.

Reply:
Good point, this text was stale (no pun intended), fixed.